source/shared/core_sqlsrv.h

```diff
@@ -1310,7 +1343,11 @@ const int SQLSRV_DEFAULT_SIZE = -1;     // size given for an output parameter th
 const unsigned int QUERY_TIMEOUT_INVALID = 0xffffffff;
 
 // special buffered query constant
+#ifndef _WIN32
```

Comment:
Is it necessary to have this defined two ways? The original caused bugs in Linux, but can this be 42 in Windows?